### PR TITLE
Pinot website

### PR DESCRIPTION
**Pull request by ChethanUK**
WIP.. 

Check https://pinot-website.now.sh/ or https://pinot-website.now.sh/docs/administration/installation/cloud/on-premises to see more about the website..

### CODE REVIEW

**Comment by snleee** [comment]
Hi, is this trying to replace http://pinot.apache.org?

FYI, `http://pinot.apache.org/` is served directly from `https://github.com/apache/incubator-pinot-site`. They will load `index.html` file. 

In that case, we will have 2 different approaches.
1. keep both source code + generated static website in `incubator-pinot-site`.
2. keep website code in `incubator-pinot`, whenever there's an update put static files to `incubator-pinot-site`.


Also, I have 2 comments:
1. can we change the color from sky-blue to red wine like color? (Since pinot is a kind of wine :) )
2. your static website generator needs a copy of all documentation. This means that we will need to keep 2 separate sources (https://apache-pinot.gitbook.io/apache-pinot-cookbook) we need to find the way to have a single source for documentation.

@kishoreg

**Comment by ChethanUK** [comment]
> 1. red wine like color

Okay, Give me hex code.


> trying to replace

Not decided yet..



> different approaches.

According to me, it will be best to use one repo for both codes and versioned docs instead of many repos like some other projects... It will be just other another folder and then just have to generate and push the content to asf-site

**Comment by snleee** [comment]
@ChethanUK `#722f37` seems to be the wine color. I also like the idea of having source code in `incubator-pinot` code base and push the static files to `incubator-pinot-site` when we have the update.

We will need to figure out if we want to migrate back to source code for documentation again. (we migrated out our doc to gitbook recently..)

**Comment by ChethanUK** [comment]
> `#722f37`

I don't like this hex personally...

Check these images: 
Main

Docs

Light mode :
LightMode

File: website/docs/administration/installation/cloud/on-premises.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
+---
```
Comment (fx19880617) [inline website/docs/administration/installation/cloud/on-premises.md]: Change filename to `on-premise.md`

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  themeConfig: {
+    navbar: {
+      hideOnScroll: true,
+      title: 'Pinot™ (Incubating)',
+      logo: {
+        alt: 'Pinot',
+        src: 'img/logo.svg',
+      },
+      links: [
+        {to: 'https://apache-pinot.gitbook.io/apache-pinot-cookbook/', label: 'Docs', position: 'right'},
```
Comment (fx19880617) [inline website/docusaurus.config.js]: link is `https://docs.pinot.apache.org/`

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        {to: 'https://issues.apache.org/jira/projects/PINOT/issues', label: 'Jira', position: 'right'},
```
Comment (fx19880617) [inline website/docusaurus.config.js]: delete jira

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        {to: 'https://cwiki.apache.org/confluence/display/PINOT', label: 'Wiki', position: 'right'},
```
Comment (fx19880617) [inline website/docusaurus.config.js]: delete wiki

File: website/docusaurus.config.js (hunk 2 above)
Comment (fx19880617) [inline website/docusaurus.config.js]: Add a Download Page after Docs.

File: website/docusaurus.config.js (hunk 4 above)
Comment (fx19880617) [inline website/docusaurus.config.js]: You can add a Blog link to `https://docs.pinot.apache.org/community-1/blogs`

File: website/docusaurus.config.js [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+        },
+        {
+          title: 'Components',
+          items: [
+            {
+              label: 'Presto',
+              to: 'https://docs.pinot.apache.org/integrations/presto',
+            },
+            {
+              label: 'PQL',
```
Comment (fx19880617) [inline website/docusaurus.config.js]: Change PQL to Superset:
```
            {
              label: 'Superset',
              to: 'https://docs.pinot.apache.org/integrations/superset',
            },
```

File: website/docusaurus.config.js [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+              to: 'docs/components/sources',
+            },
+            {
+              label: 'ThirdEye',
+              to: 'https://docs.pinot.apache.org/integrations/thirdeye',
+            },
+            {
+              label: 'PowerBI',
```
Comment (fx19880617) [inline website/docusaurus.config.js]: delete power BI

File: website/docusaurus.config.js [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+      logo: {
+        alt: 'Apache Pinot™ - Incubating',
+        src: 'img/logo.svg',
+        href: 'https://pinot.apache.org/',
+      },
+      copyright: `Copyright © ${new Date().getFullYear()} The Apache Software Foundation.`,
+    },
+    googleAnalytics: {
+      // TODO
+      trackingID: 'TEMP',
```
Comment (fx19880617) [inline website/docusaurus.config.js]: Please refer to `https://github.com/apache/incubator-pinot-site/pull/16/files`
Is this the tracking id 'UA-157446650-1'?

File: website/docusaurus.config.js [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+    },
+    algolia: {
+      apiKey: 'f3cde09979e469ad62eaea4e115c21ea',
+      indexName: 'apache_pinot',
+      algoliaOptions: {}, // Optional, if provided by Algolia
+    },
```
Comment (fx19880617) [inline website/docusaurus.config.js]: Can we also add twitter tag manager? https://github.com/apache/incubator-pinot-site/pull/17/files

File: website/scripts/publish_site.sh [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+(
+
+  cd $ROOT_DIR
+  rm -rf $SITE_TMP
+  mkdir $SITE_TMP
+  cd $SITE_TMP
+
+  git clone "https://$GH_TOKEN@$ORIGIN_REPO" .
+  git config user.name "Pinot Site Updater"
+  git config user.email "dev@pinot.incubator.apache.org"
```
Comment (fx19880617) [inline website/scripts/publish_site.sh]: dev@pinot.apache.org

File: website/docusaurus.config.js (hunk 8 above)
Comment (fx19880617) [reply website/docusaurus.config.js]: if not intuitive, you can skip it this time.

Comment (ChethanUK) [reply website/docusaurus.config.js]: Will add this later. Search not required now..

File: website/README.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+```
+$ GIT_USER=CHETHANUK USE_SSH=true yarn deploy
```
Comment (fx19880617) [inline website/README.md]: this line should be different for everyone?

File: website/README.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+# Website
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```
Comment (fx19880617) [inline website/README.md]: Could you provide a link or instructions to install `yarn`?

File: website/scripts/publish_site.sh [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+  git checkout asf-master
+
+  # Clean content directory
+  rm -rf $SITE_TMP/content/
+  mkdir $SITE_TMP/content
+
+  # Copy the generated directory to asf folder
+  cp -r $WORK_DIR/* $SITE_TMP/content
+)
+
```
Comment (fx19880617) [inline website/scripts/publish_site.sh]: remove tailing lines

Comment (ChethanUK) [reply website/scripts/publish_site.sh]: Okay!

File: website/src/pages/index.js [hunk 13]
```diff
@@ -0,0 +1,297 @@
...
+        SQL like language that supports selection, aggregation, filtering, group by, order by, distinct queries on data
+      </>
+    ),
+  },
+  {
+    title: 'Hybrid tables',
+    icon: 'list',
+    description: (
+      <>
+        Consist of of <Link to="/img/dynamic-table.png">both offline and realtime table</Link>. Use realtime table only to cover segments for which offline data may not be available yet
```
Comment (fx19880617) [inline website/src/pages/index.js]: does this link work? `/img/dynamic-table.png`

Comment (ChethanUK) [reply website/src/pages/index.js]: Yes

**Comment by fx19880617** [comment]
This is awesome! Many  thanks for your contribution @ChethanUK !

**Pull request merged**